Source: written by Pierre Lejeune, for Exherbo
Upstream: See http://wiki.audacityteam.org/wiki/Known_Issues#Active_issues
Reason: Fix compilation of FFmpeg plugin
--- src/FFmpeg.cpp.orig	2010-03-30 13:28:54.000000000 +0200
+++ src/FFmpeg.cpp	2010-10-26 20:56:19.705092448 +0200
@@ -204,7 +204,7 @@
    return ret;
 }
 
-static int ufile_write(URLContext *h, unsigned char *buf, int size)
+static int ufile_write(URLContext *h, const unsigned char *buf, int size)
 {
    return (int) ((wxFile *) h->priv_data)->Write(buf, size);
 }
